SecretSanta/src/SecretSanta.Api/Controllers/UserController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using SecretSanta.Domain.Interfaces;
+using SecretSanta.Domain.Models;
+
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace SecretSanta.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UserController : ControllerBase
+    {
+        private readonly IUserService _UserService;
+
+        public UserController(IUserService userService)
+        {
+            _UserService = userService ?? throw new ArgumentNullException(nameof(userService));
+        }
+
+        // GET api/User/5
+        [HttpGet("{id}")]
+        public ActionResult<DTO.User> FindUser(int id)
+        {
+            if(id <= 0)
+            {
+                return NotFound();
+            }
+
+            DTO.User user = new DTO.User(_UserService.Find(id));
+
+            return user;
```

Comment:
This is a styling comment, but you could reduce this to a single line
```suggestion
            return new DTO.User(_UserService.Find(id));
```